config/checkstyle_non_main_files_checks.xml

```diff
@@ -24,6 +24,15 @@
     <message key="regexp.filename.mismatch"
       value="CI folder should only contain executables files"/>
   </module>
+  <module name="RegexpOnFilename">
+    <property name="id" value="regexpOnGithubWorkflows"/>
+    <property name="folderPattern" value="[\\/].github[\\/]workflows([\\/]|$)"/>
+    <property name="fileNamePattern" value="^([a-z][a-z0-9]*)(-[a-z0-9]+)*\..*$"/>
```

Comment:
`-` should be sanitized since it is a character as is and we don't want it to accidentally picked up as something else.
`-` => `\-`

Reply:
Done.